api.planx.uk/server.ts

```diff
@@ -188,6 +189,7 @@ app.use(passport.session());
 app.use(urlencoded({ extended: true }));
 
 app.use(authRoutes);
+app.use("/team", teamRoutes);
```

Comment:
I'd expect that we want `useJWT` here similar to admin endpoints defined like this ? 
```
app.use("/admin", useJWT);
app.get("/admin/feedback", downloadFeedbackCSV);
```

Reply:
Good catch - it's actually still there, I've just moved the responsibility to the routes module.

`router.use(AuthMiddleware.useJWT);` in `modules/team/routes.ts`